src/us/kbase/workspace/test/kbase/HandleAndShockTest.java

```diff
@@ -169,36 +169,28 @@ public static void setUpClass() throws Exception {
 		WS_OWNED_SHOCK = new BasicShockClient(shockURL, HANDLE_MNGR_TOKEN);
 		SHOCK_CLIENT_1 = new BasicShockClient(shockURL, t1);
 
-		MYSQL = new MySQLController(
-				TestCommon.getMySQLExe(),
-				TestCommon.getMySQLInstallExe(),
-				Paths.get(TestCommon.getTempDir()));
-		System.out.println("Using MySQL temp dir " + MYSQL.getTempDir());
-		
 		HANDLE = new HandleServiceController(
-				WorkspaceTestCommon.getPlackupExe(),
-				WorkspaceTestCommon.getHandleServicePSGI(),
-				WorkspaceTestCommon.getHandleManagerPSGI(),
-				"user3",
-				MYSQL,
+				MONGO,
 				shockURL.toString(),
-				HANDLE_MNGR_TOKEN,
```

Comment:
Why change the token here? I didn't want to mix up the test tokens with the handle service token.

Reply:
👍